tests/components/wmspro/responses.py

```diff
@@ -0,0 +1,243 @@
+"""Example JSON responses for the wmspro tests."""
+
+import json
+
+
+def example_config_test():
+    """Return JSON configuration taken from official documentation."""
+    return json.loads("""
+{
+    "command": "getConfiguration",
+    "protocolVersion": "1.0.0",
+    "destinations": [
+        {
+            "id": 17776,
+            "animationType": 0,
+            "names": [
+                "Küche",
+                "",
+                "",
+                ""
+            ],
+            "actions": [
+                {
+                    "id": 0,
+                    "actionType": 0,
```

Comment:
Please move these to `fixtures` json files and use `load_json_object_fixture` to load them in